common/io/Serial.cpp

```diff
@@ -57,6 +58,7 @@ using std::string;
 
 namespace {
 
+#ifdef UUCP_LOCKING
```

Comment:
I'm not sure of the benefit of not defining these if we're not in UUCP mode, it looks like they should work in all cases.

Reply:
Ok, I removed the ifdefs, which reduces the noise a bit.